spec.md

```diff
@@ -438,27 +440,31 @@ extension authors SHOULD check the
 extensions - not just for possible name conflicts but for extensions that might
 be of interest.
 
-Each specification that defines how to serialize a CloudEvent will define how
-extension attributes will appear.
+#### Guidance for binding specifications
+
+Each specification that defines how to serialize a CloudEvent MUST define how to
+serialize extension attributes. The following are some guidelines.
 
-Extension attribtue MUST be serialized using the same general pattern as all
-CloudEvents context attributes. For example, in binary HTTP, that means they
-MUST appear as HTTP headers with the `ce-` prefix. The specification of an
-attribute MAY define a secondary serialization where the data is duplicated in
-some other location within the message.
+Unless there is reason to do otherwise, extension attributes are serialized in
+the same way as standard attributes. For example the HTTP binding encodes both
+as HTTP headers with a "ce-" prefix. There can be exceptions, for example the
+`datacontentype` attribute is stored in the standard HTTP `Content-Type` header
+so that it is visible to other HTTP processes.
 
-In cases where a secondary serialization is defined, the extension specification
-MUST also state what a receiver of the CloudEvent is to do if the data differs
-between those two serialization locations. Additionally, senders need to be
-prepared for intermediaries, and receivers, to not know about their extension
-and therefore the specialized serialization version will most likely not be
-processed as a CloudEvent extension attribute.
+A binding MAY serialize the same attribute value to multiple locations in a
+message. For example HTTP tracing repeats some attribute values in "ce-" headers
+and in other headers examined by tracing agents. Such bindings MUST specify how
+receivers should handle conflicting values for duplicated attributes, and how
+senders should format messages to ensure that the desired attribute value is
+transmitted.
 
 Many protocols support the ability for senders to include additonal metadata,
```

Comment:
Many protocols support the ability for senders to include additonal metadata,
for example as HTTP headers. While a CloudEvents receiver is not mandated to
process and pass them along, it is RECOMMENDED that they do so via some
mechanism that makes it clear they are non-CloudEvents metadata.

Reply:
Basically yes and I think it was driven by those duplicate serialized extensions.  So, in the tracing case we were thinking that it would appear as `ce-traceparent` and HTTP header `traceparent`, and if they had different values we wanted to encourage implementations to expose both to the receiving app so they could decide what to do about the diff.  If they only ever saw the ce- one (and couldn't even get to the other one) then we're kind of setting people up for a situation where they're forced to only deal with one of the two.